packages/expo-notifications/ios/EXNotifications/Permissions/EXUserFacingNotificationsPermissionsRequester.m

```diff
@@ -83,25 +83,39 @@ - (void)requestPermissionsWithResolver:(EXPromiseResolveBlock)resolve rejecter:(
 - (void)requestPermissions:(NSDictionary *)permissions withResolver:(EXPromiseResolveBlock)resolve rejecter:(EXPromiseRejectBlock)reject
 {
   UNAuthorizationOptions options = UNAuthorizationOptionNone;
-  if ([permissions[@"allowAlert"] boolValue]) {
+  if ([permissions.allKeys containsObject:@"allowAlert"] &&
```

Comment:
there's lots of repetition in these checks, would be nicer to extract it into a reusable method

Reply:
Refactored this into the Swift Record struct.